frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/HiveNotification.tsx

```diff
@@ -74,6 +86,14 @@ export function HiveNotification() {
                 title={
                     <Fragment>
                         {t(`provision.notification.${cluster?.status}`)}
+                        {isHybrid && 
+                            <LogsDownloadButton
+                                id="cluster-logs-button"
+                                agentClusterInstall={cluster}
+                                variant={ButtonVariant.link}
+                            />                            
+                        }
+                        {!isHybrid &&
```

Comment:
This part is not needed since we don't render the alert for hybrid clusters at all.